bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/service/ItcServiceImpl.scala

```diff
@@ -26,8 +29,24 @@ class ItcServiceImpl extends ItcService {
 
   def calculate(source: SourceDefinition, obs: ObservationDetails, cond: ObservingConditions, tele: TelescopeDetails, ins: InstrumentDetails): Result = try {
 
-    if (obs.getMethod.isImaging)  calculateImaging(source, obs, cond, tele, ins)
-    else                          calculateSpectroscopy(source, obs, cond, tele, ins)
+    // Get the SED data from an aux file. For now we can assume that the ITC service is running on the same
+    // machine as the database (localhost). In case this setup changes, we need to change this here, too.
+    def auxFileDistribution(id: String, name: String): SpectralDistribution = {
+      val programId     = SPProgramID.toProgramID(id)
+      val spectrumBytes = new AuxFileClient("localhost", 8443).fetchToMemory(programId, name)
+      val spectrum      = new String(spectrumBytes)
```

Comment:
The is the first auxfile type where we convert the data to/from Strings. Do we need to worry about character encodings?  How are the files created?